doc/source/user_guide/deployment_juicefs_on_kubernetes.rst

```diff
@@ -0,0 +1,466 @@
+.. _deployment_juicefs_on_kubernetes:
+
+=====================
+JuiceFS on Kubernetes
+=====================
+
+Xorbits is able to utilize `JuiceFS <https://juicefs.com/en/>`_, distributed POSIX file system that can be easily integrated with Kubernetes to provide persistent storage, as one of the storage backend.
+
+Prerequisites
+-------------
+Xorbits
+~~~~~~~~~~~~~~
+Install Xorbits on the machine where you plan to deploy Kubernetes with JuiceFS.
+Refer to :ref:`installation document <installation>`.
+
+Metadata Storage
+~~~~~~~~~~~~~~
+JuiceFS decouples data and metadata. Many databases are supported. See `How to Set Up Metadata Engine <https://juicefs.com/docs/community/databases_for_metadata>`_ and choose an appropriate metadata storage.
+
+In our example here, we select ``Redis`` as our metadata storage.
+
+Follow `Configuring Redis using a ConfigMap <https://kubernetes.io/docs/tutorials/configuration/configure-redis-using-configmap/>`_ and create a pod inside default namespace.
+
+You should set its maxmemory as 50mb since 2mb in the example is too small.
+
+Make sure redis pod is running:
+
+.. code-block:: bash
+
+    $ kubectl get po redis
+    NAME    READY   STATUS    RESTARTS    AGE
+    redis   1/1     Running   0           6d6h
+
+..
+
+
+Check redis pod's IP address, cpu limits and requests. In this example, IP for redis is 172.17.0.8.
+
+.. code-block:: bash
+
+    $ kubectl describe po redis
```

Comment:
Here you just want to verify redis' IP. You can restrict the kubectl output.Now this takes up too much space  on the page.